### ISSUE TITLE
Returning all results

### ISSUE THREAD

**Issue by Ben-Baert** [coleifer/walrus#19]
Is there an easy way to return all results if no partial phrase is entered? Something like

```
>>> phrases = ['aa bb', 'aa cc', 'bb cc', 'bb aa cc', 'cc aa bb']
>>> for phrase in phrases:
...     ac.store(phrase)
>>> ac.search('')
['aa bb', 'aa cc', 'bb cc', 'bb aa cc', 'cc aa bb']
```

**Comment by coleifer** [comment]
Hmm, I hadn't really thought about that use-case. At this time, no, there is not an easy way to list all possible results.

**Comment by coleifer** [comment]
This has been implemented in ce33ebe, but there's a caveat. Since this is autocomplete, if nothing has been typed, I think it's best not to display any results and not run the risk of sending a massive amount of data down the wire.

I've added two new APIs for listing all the titles and all the data stored in the autocomplete index. These methods are documented here:

* http://walrus.readthedocs.io/en/latest/api.html#walrus.Autocomplete.list_data
* http://walrus.readthedocs.io/en/latest/api.html#walrus.Autocomplete.list_titles
